DesiMock

```diff
@@ -0,0 +1,195 @@
+from astropy.io import fits
```

Comment:
Is the name of this file simply `DesiMock` ?  
It should have a .py extension if it is a module.

Reply:
Sorry, this is an oversight on my part... I‘ll change it.